assets/ts/stop/components/DeparturesWithBadge.tsx

```diff
@@ -57,16 +48,23 @@ const DeparturesWithBadge = ({
   if (!(priorityBadge || secondaryBadge)) return <>{children}</>;
   const displayBadge = (priorityBadge || secondaryBadge)!;
 
+  // if the priority badge is present and there is only one departure, show the badge
+  // this is because we check for predictions and return a single time list if there are none
   return (
     <>
-      {priorityBadge ? (
-        <div className="font-helvetica-neue fs-14" style={{ float: "right" }}>
-          See alternatives
+      {priorityBadge && timeListLength === 1 ? (
```

Comment:
There's some logic involved in the generation of `priorityBadge` that attempts to weight the number of predictions available against the particular type of alerts – have you looked at `isSuppressiveAlert` to see whether modifying that would better fit our use case?